easybuild/easyconfigs/i/intltool/intltool-0.51.0-GCCcore-6.4.0-Perl-5.26.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'intltool'
+version = '0.51.0'
+versionsuffix = '-Perl-5.26.0'
+
+homepage = 'http://freedesktop.org/wiki/Software/intltool/'
+description = """intltool is a set of tools to centralize translation of
+ many different file formats using GNU gettext-compatible PO files."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://launchpad.net/intltool/trunk/%(version)s/+download/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('XML-Parser', '2.44_01', versionsuffix),
+]
+
+# intltool bug with Perl 5.26
+# https://bugs.launchpad.net/intltool/+bug/1696658
+# This patch ist from:
+# https://github.com/Alexpux/MSYS2-packages/blob/master/intltool/perl-5.22-compatibility.patch
```

Comment:
no need to include these comments here, they're already in the patch file itself?

Reply:
Maybe just something like "fix regex error with perl 5.26.0" at this point, so one can understand instantly what the patch is for when reading the easyconfig?

Reply:
If people want to figure out what the patch does exactly, they can look into the patch file.

The patch filename could give a good first hint.